syndesis/src/setupProxy.js

```diff
@@ -0,0 +1,43 @@
+/* tslint:disable */
+const talkback = require('talkback');
+const proxy = require('http-proxy-middleware');
+const filenamify = require('filenamify');
+const config = require('../public/config.json');
+
+// Disable self-signed certificate check
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
+
+const backendName = filenamify(process.env.BACKEND);
+
+const server = talkback({
+  host: process.env.BACKEND,
+  port: 556,
```

Comment:
Can we pick a port > 1024 for this?  On Linux at least you can't open ports < 1024 unless you're root.

Reply:
Tryin' 5556 locally for now.

Reply:
Ah good point, yes it can be whatever we want it to be!